driver/src/test/scala/org/mongodb/scala/model/vault/ClientEncryptionSpec.scala

```diff
@@ -48,23 +41,27 @@ class ClientEncryptionSpec extends FlatSpec with Matchers with MockFactory {
   it should "call createDataKey" in {
     val kmsProvider = "kmsProvider"
     val options = DataKeyOptions()
-    wrapped.expects(Symbol("createDataKey"))(kmsProvider, options, *).once()
-    clientEncryption.createDataKey(kmsProvider, options).subscribe(observer[BsonBinary])
+
+    wrapped.expects(Symbol("createDataKey"))(kmsProvider, *).once()
+    clientEncryption.createDataKey(kmsProvider)
+
+    wrapped.expects(Symbol("createDataKey"))(kmsProvider, options).once()
+    clientEncryption.createDataKey(kmsProvider, options)
   }
 
   it should "call encrypt" in {
     val bsonValue = BsonString("")
     val options = EncryptOptions("algorithm")
-    wrapped.expects(Symbol("encrypt"))(bsonValue, options, *).once()
+    wrapped.expects(Symbol("encrypt"))(bsonValue, options).once()
 
-    clientEncryption.encrypt(bsonValue, options).subscribe(observer[BsonBinary])
+    clientEncryption.encrypt(bsonValue, options)
   }
 
   it should "call decrypt" in {
     val bsonBinary = BsonBinary(Array[Byte](1, 2, 3))
-    wrapped.expects(Symbol("decrypt"))(bsonBinary, *).once()
```

Comment:
What did the `*` mean?

Reply:
`*` means match any parameter, we used it match the callbacks.